### ISSUE TITLE
mising latte panda all of them

### ISSUE THREAD

**Issue by jusrose** [DFRobot/Fritzing-library#13]
I can't find the latte panda original, alpha or delta. is it even in there
if not plz add it

**Comment by bdz69** [comment]
Bump!

**Comment by dmusker** [comment]
Yes, I need this too.  It must exist because it is used in the DFRobot documentation examples.  Does anyone have this?  Please upload.
